shared/iot/develop/encrypt-media-streams/project-implementation/linux-c.mdx

```diff
@@ -1,4 +1,102 @@
 <PlatformWrapper platform="linux-c">
 
```

Comment:
The whole "Encrypt media streams" document does not show in the local build. Is this supposed to be so?